RELEASE/scripts/sl_ascend.ash

```diff
@@ -9738,11 +9738,11 @@ boolean LX_steelOrgan()
 	else if(get_property("questM10Azazel") == "finished")
 	{
 		print("Considering Steel Organ consumption.....", "blue");
-		if((item_amount($item[Steel Lasagna]) > 0) && (fullness_left() >= 5))
+		if((item_amount($item[Steel Lasagna]) > 0) && (fullness_left() >= $item[Steel Lasagna].fullness))
 		{
 			eatsilent(1, $item[Steel Lasagna]);
 		}
-		if((item_amount($item[Steel Margarita]) > 0) && ((my_inebriety() <= 5) || (my_inebriety() >= 12)))
+		if((item_amount($item[Steel Margarita]) > 0) && (inebriety_left() >= $item[Steel Margarita].inebriety - 5) && (my_inebriety() <= inebriety_limit()))
```

Comment:
I think the original intent with the check for `my_inebriety() <= 5` was to avoid getting over 10 inebriety with it so it doesn't spoil the billiards room, with the `my_inebriety() >= 12` check being to say "eh, we're already past a good amount of inebriety anyway." Maybe something like 
```
(item_amount($item[Steel Margarita]) > 0) && (inebriety_left() >= $item[Steel Margarita].inebriety - 5) && ((my_inebriety() + $item[Steel Margarita].inebriety <= 10) || (my_inebriety() >= 12))
```
would be better for the condition? The `my_inebriety() <= inebriety_limit()` check is kind of redundant with the `inebriety_left() >= $item[Steel Margarita].inebriety - 5` check, I think.

Reply:
We could also check if the billiards room is complete and ignore the `my_inebriety() + $item[Steel Margarita].inebriety <= 10` part if so, just to be thorough, although that's probably overkill...

Reply:
Wouldn't that be good for paths that don't use the billiards room though? making it more future proof?

Reply:
Yeah, I'll make it consider billiards room completion. TODO